### ISSUE TITLE
Material and Project won't save

### ISSUE THREAD

**Issue by kazsud** [BoundingBoxSoftware/Materialize#7]
App folder is on desktop on a Windows 10 Pro machine.

OS Build 17134.345

Very easy to use.

**Comment by roymina** [comment]
same problem, but when i select another place, all textures and project can save

**Comment by PJuniszewski** [comment]
Launch the program as admin and try to save on desktop. Worked on my Windows 10 Pro

**Comment by RichardR01** [comment]
This is a still ongoing issue. I opened Materialize as admin. It won't save to the folder I tell it to. However it does save to my Downloads folder. The folder I want to save it to has full permissions so I don't understand why it won't save there.

**Comment by mariamont** [comment]
No puedo guardar los proyectos , ya que no me aprece la barra para crear una nueva carpeta y guardar las texturas. Lo desisntale e instale varias veces pero no me aparece, no si esta mal configurado y alguien me puede ayudar.
![Captura de pantalla (19)](https://user-images.githubusercontent.com/82049202/113749460-125f2c80-96e0-11eb-98a5-a3578bf9aa03.png)

**Comment by EdchaSw** [comment]
Hola @mariamont en la esquina inferior derecha de esa ventana que muestras tienes el triangulito donde si lo tomas con el mouse puedes agrandar la ventana y te mostrará los campos y botones donde guardar el proyecto, Saludos

**Comment by lilpoup** [comment]
![trim](https://user-images.githubusercontent.com/99398241/153387987-94264086-ba4c-4815-b514-0839b263daea.png)
you need to click set as height map and than click s to save

**Comment by lovely1987** [comment]
hola alguien pudo solucionarlo? no me da la opción de Save

**Comment by Poopodome** [comment]
For anyone still having this issue. Try hitting the S button next to each of your maps and selecting an image file to use as a saving point instead of a folder. Idk why, but that seems to work for me to be able to save all of my maps.

**Comment by michaelccc** [comment]
I'm also having this issue, as you mentioned the saving to another image file worked. Really kills the usability of the application tho.

**Comment by WilSkywalk3r** [comment]
Hey guys! Does anyone with the same problem? I just figured it out! To fix it you just have to select the wanted fold and type a name for  the files.
When you type the Filename it will enable the Select bottom.

Hope this help you   guys! 

![image](https://github.com/BoundingBoxSoftware/Materialize/assets/151659062/75548a48-0361-4234-9956-e8361bc97d4d)

**Comment by DiggyDre** [comment]
Yeah, this thing is not saving out anything. I give it a name, and choose where I want to save... the window goes away but no files are created. It's a bummer because I was actually able to get something to tile better in here than I was able to in substance sampler! But it doesn't matter... If I can't get the textures out of this thing, then it makes this one useless for my workflow.

**Comment by mcleantom** [comment]
I managed to save a photo by clicking on an existing photo in a folder and overwriting it

**Comment by Pixelsplasher** [comment]
### [SOLVED] In order to save the project successfully, you must type **.mtz** to the end of your filename or select an existing .mtz file to overwrite or to copy its filename and save in another location.

**Comment by DeXxik62** [comment]
I had this problem too and the solution is simple. You need to add an extension (.jpg, .bmp, .tiff) to the name. When you want a JPG in the app you click and then put save over "S". 
yourname.jpg

**Comment by juancy1234** [comment]
to save all the images only click one of all "s" botton of the materials

<img width="912" height="275" alt="Image" src="https://github.com/user-attachments/assets/609bcacb-9e7c-4746-bc3d-67dd8f2cec63" />
